pkg/pillar/cmd/diag/diag.go

```diff
@@ -343,7 +343,7 @@ func handleDNSModify(ctxArg interface{}, key string, statusArg interface{}) {
 	}
 	log.Infof("handleDNSModify for %s", key)
 	// Since we report test status we compare all fields
```

Comment:
Need to take this comment into account.
It is fine to supress the ping, but we want diag to be able to report the overall State of the DPC.

Reply:
So, should I revert this back to original code?

Reply:
Will revert back to original code.